include/ofi_shm.h

```diff
@@ -285,8 +285,8 @@ struct smr_sar_msg {
 
 OFI_DECLARE_CIRQUE(struct smr_cmd, smr_cmd_queue);
 OFI_DECLARE_CIRQUE(struct smr_resp, smr_resp_queue);
-SMR_DECLARE_FREESTACK(struct smr_inject_buf, smr_inject_pool);
-SMR_DECLARE_FREESTACK(struct smr_sar_msg, smr_sar_pool);
+SMR_DECLARE_FREESTACK(smr_inject_pool);
+SMR_DECLARE_FREESTACK(smr_sar_pool);
```

Comment:
Now that the freestack is generic I don't think you should need these declarations

Reply:
We will remove these